release-notes/current.yaml

```diff
@@ -1,29 +1,5 @@
-date: Pending
-
-# Changes that are expected to cause an incompatibility with previous versions, such as deletions or modifications to existing APIs.
-breaking changes: |
-  - Add a breaking change here
-
-# Updates addressing vulnerabilities, security flaws, or compliance requirements.
-security updates: |
-  - Add a security update here
-
-# New features or capabilities added in this release.
-new features: |
-  - Add a new feature here
-
-# Fixes for bugs identified in previous versions.
-bug fixes: |
-  - Add a bug fix here
-
-# Enhancements that improve performance.
-performance improvements: |
-  - Add a performance improvement here
-
-# Deprecated features or APIs.
-deprecations: |
-  - Add a deprecation here
-
-# Other notable changes not covered by the above sections.
-Other changes: |
-  - Add other changes here
+date: Oct 23, 2022
+changes:
+  - area: api
+    change: |
+      Support setting trusted CIDRs
```

Comment:
We're adopting a new release format. This change can be added to the new features section.